### ISSUE TITLE
Eraser doesn't erase if you drag it over objects too fast

### ISSUE THREAD

**Issue by PMKielstra** [edemaine/cocreate#46]
Draw a line.  Select the eraser tool.  With the mouse down, move the eraser across the line very quickly.  The line is not erased.

**Comment by edemaine** [comment]
Indeed, currently collision is detected only at the sample points, without any interpolation in between. Linear interpolation would make sense.  I'm hoping there's a DOM query for this type of intersection...

**Comment by ni-tro** [comment]
+1 on this issue. Using a pen on a tablet, the eraser is even slower.
For me this makes using cocreate a lot more time-consuming and has the effect that I rather use a different tool for scribbling on my own and later recreate the important parts in a shared Cocreate whiteboard...

**Comment by edemaine** [comment]
A workaround for now is to use the select tool (drag to rectangular select) and then pressing the Delete key. (But that doesn't work on a tablet-only device.)

**Comment by edemaine** [comment]
I just implemented an improvement to the eraser object detection (looking at all coalesced events instead of just the final event in a sequence) that should help &mdash; hopefully a lot and perhaps especially when using a pen on a tablet &mdash; though there's more that could be done after #87.  Let me know if you see improvements in your environment, @ni-tro.

**Comment by ndattani** [comment]
I only started using this within the last month, so your improvement from 1 year ago (mentioned in your last comment) would already have been implemented, but I have a very hard time using the eraser tool! Often it doesn't do anything, and often it's slow (and causes my CPU to start getting overworked). I instead use the select tool to make a rectangle and then press delete, but unfortunately, lots of other objects that I do not want deleted (which are not even in the rectangle!) get deleted along with what's inside the rectangle, which is quite inconvenient.
